devito/finite_differences/finite_difference.py

```diff
@@ -117,7 +53,7 @@ def first_derivative(expr, dim, fd_order=None, side=centered, matvec=direct,
 
     This is also more easily obtainable via:
 
-    >>> (f*g).dx
+    >>> (f*g).dx.evaluate
     -f(x, y)*g(x, y)/h_x + f(x + h_x, y)*g(x + h_x, y)/h_x
 
     The adjoint mode
```

Comment:
nitpicking: we may shown again the shortcut notation for this via .transpose

Reply:
Done